src/utils/entitySpec.js

```diff
@@ -74,6 +74,10 @@ function normalize(value, provider, property) {
 }
 
 export default class EntitySpec {
+  static isPath(path) {
+    return path.match(/\/*([^/]+)\/([^/]+)\/([^/]+)\/([^/]+)\/?([^/]+)?(\/pr\/.+)?/)
+  }
+
   static fromPath(path) {
```

Comment:
can we update fromPath to return null if there is no match?

So we don't have to run the same regex twice?